### DISCUSSION TITLE
Asking for Username/Password on Truenas

### DISCUSSION THREAD

**Discussion by JamieLee2k** [AdguardTeam/AdGuardHome#7539] [Q&A]
I installed adguard-home on Truenas Scale and I remember that when I installed it it asked for a username/password which I kept simple in case I forgot it, when I click the WebIU on truenas apps it sometimes just goes in without a username/password but now it's asking me for it and it wont accept the one I put in, so how do I access it without removing it all and starting again?

**Comment by schzhn** [comment]
https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#password-reset

1. Generate a hash for your new password. You can use `mkpasswd` or any online password hashing tool.

1. Stop AdGuard Home.

1. Edit the `users.password` field in the configuration file e.g. `/usr/local/bin/AdGuardHome.yaml`, or find the file using the `find / -name AdGuardHome.yaml`. 

1. Start AdGuard Home:

Related #6865
